build/windows/nanoserver/Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM microsoft/nanoserver
+ARG SOURCE_TAG=latest
 
-COPY dist /
+FROM microsoft/nanoserver:${SOURCE_TAG}
+
+USER ContainerAdministrator
 
-VOLUME C:\\data
```

Comment:
Why did you choose to remove the default VOLUME statement?

Reply:
I find out on Windows Server, version 1803 where it tested these images using Docker process isolation mode that if VOLUME statement is on Dockerfile Portainer does not see C:\Data folder and wants create that but Windows things that it is there and folder creating fails. That that fails whole Portainer start process. Without that line it works correctly.

Reply:
@seal-ss are you aware of any issues using the `VOLUME` statement in Windows server 1803?

Reply:
Hmm. I did now some more testing and found that ```USER ContainerAdministrator``` line actually fixes also this issue.
If that line is not on file and VOLUME statement is starting portainer gives error like this:
```2018/07/11 10:30:14 mkdir C:\data: Cannot create a file when that file already exists.```

So I will restore that line to file.